gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -127,9 +133,17 @@ final void setManifestExtensions(Map<String, Object> manifestExtensions) {
         return ignoredProductIds;
     }
 
-    @InputFiles
-    final Property<Configuration> getProductDependenciesConfig() {
-        return productDependenciesConfig;
+    @Input
+    final Set<String> getProductDependenciesConfig() {
```

Comment:
Would like to try and avoid making any network calls at configuration time if poss

Reply:
this won't happen at configuration time